File: frontend/src/plugins/simulators/phone/manifest.ts
import { lazy } from 'react'
import { Phone } from 'lucide-react'
import type { PluginManifest } from '../../types'

export const manifest: PluginManifest = {
  code: 'phone',
  name: 'plugins.phone.name',
  description: 'plugins.phone.description',
  category: 'simulator',
  icon: Phone,
  defaultColor: 'green',
  supportedFormats: ['audio'],
  defaultEnabled: false,
  comingSoon: true,
  sortOrder: 8,
  injectTypes: [],
  playerRoute: '/phone',
  PlayerPage: lazy(() => import('./PlayerPage')),
  EventReceiver: lazy(() => import('./EventReceiver')),
  PreviewPopup: lazy(() => import('./PreviewPopup')),
  configSchema: null,
}

File: backend/app/plugins/simulators/phone/manifest.py
from app.plugins.base import PluginCategory, PluginManifest

manifest = PluginManifest(
    code="phone",
    name="Telephone",
    name_en="Phone",
    description="Simulation d'appels telephoniques",
    category=PluginCategory.SIMULATOR,
    icon="Phone",
    default_color="green",
    supported_formats=["audio"],
    default_enabled=False,
    coming_soon=True,
    sort_order=8,
    inject_types=[],
    router_module=None,
    router_prefix=None,
)
